velox/dwio/parquet/tests/writer/CMakeLists.txt

```diff
@@ -12,7 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-add_executable(velox_parquet_writer_sink_test SinkTests.cpp)
+add_executable(velox_parquet_writer_sink_test SinkTests.cpp
+                                              ../ParquetWriterTestBase.cpp)
```

Comment:
Hmm.... why do we include a file from the parent directory? This is an anti-pattern.

Reply:
Fixed by moving them to the test/writer